src/main/resources/static/_shared.yml

```diff
@@ -2444,12 +2444,27 @@ components:
             - application
             - summaryData
     Cas2Assessment:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        nacroReferralId:
+          type: string
+        assessorName:
+          type: string
+      required:
+        - id
+    UpdateCas2Assessment:
```

Comment:
Can we not get away with having one object to create and update an assessment?  i.e. the same object is passed to  `POST` and `PUT` if that makes sense?

Reply:
we will not have `POST` endpoint as they are created by the service when the Application is submitted rather than by an HTTP request. The model to Update currently doesn't have an `id`, and the fields for `nacroReferralId` and `assessorName` are required, whereas they are not for the `Cas2Assessment` model, and it's likely to diverge later, and it means we have an object with a single responsibility of updating an Assessment.

Reply:
OK, yes `createAssessment()` is called after submitting the application.

I think whether you are creating or updating something, ultimately it could be the same object that is passed around; and that the object could have an mandatory `assessmentId` and nullable `nacroReferralId` and `assessorName` fields.  Perhaps this is because we are making fields mandatory in the front end and this is an Open API restriction.  I'll figure it out..